src/vendedores.test.ts

```diff
@@ -30,5 +76,95 @@ describe("Vendedores", () => {
         expect(viajante.puedeTrabajarEn(sierra)).toBeFalsy();
       });
     });
+    describe("Comercio Corresponsal", () => {
+      it("en una ciudad donde tenga sucursal", () => {
+        expect(comercioCorresponsal.puedeTrabajarEn(sierra)).toBeTruthy();
+      });
+
+      it("en una ciudad donde NO tenga sucursal", () => {
+        expect(comercioCorresponsal.puedeTrabajarEn(rosario)).toBeFalsy();
+      });
+    });
+  });
+
+  describe("Versatilidad", () => {
+    it("menos de 3 certificaciones no es versátil", () => {
+      expect(vendedorFijo.esVersatil()).toBeFalsy();
+    });
+    it("3 o más certificaciones pero todas del mismo tipo", () => {
+      expect(viajante.esVersatil()).toBeFalsy();
+    });
+    it("3 o más certificaciones, al menos 1 de producto, al menos 1 de NO producto", () => {
+      expect(comercioCorresponsal.esVersatil()).toBeTruthy();
+    });
+  });
+
+  describe("Firmeza", () => {
+    it("menos de 30 puntos no es firme", () => {
+      expect(vendedorFijo.esFirme()).toBeFalsy();
+    });
+    it("30 puntos es firme", () => {
+      expect(viajante.esFirme()).toBeTruthy();
+    });
+    it("más de 30 puntos es firme", () => {
+      expect(comercioCorresponsal.esFirme()).toBeTruthy();
+    });
+  });
+  describe("Influencia", () => {
+    describe("Vendedor Fijo", () => {
+      it("si es vendedor fijo no es influyente", () => {
+        expect(vendedorFijo.esInfluyente()).toBeFalsy();
+      });
+    });
+    describe("Viajante", () => {
+      it("la población de las provincias es superior a 10.000.000", () => {
+        expect(viajante2.esInfluyente()).toBeTruthy;
```

Comment:
Ojo, que al no ponerle los `()` al método lo que sucede es que este test no falla nunca - porque no se invoca el `toBeTruthy` que es el que hace el chequeo.
```suggestion
        expect(viajante2.esInfluyente()).toBeTruthy();
```